scripts/client_test/cmds/artifacts_cmd.py

```diff
@@ -297,6 +327,29 @@ def __init__(
     ) -> None:
         super().__init__("runtime")
 
+    @staticmethod
```

Comment:
the client_test is designed to call the client by the terminal command line , not call the cli.py function. cc @goldenxinxing

Reply:
yes, script is independent of the client source code

Reply:
This relys on `-o json` being implemented in bundle building process.